src/entries/popup/pages/home/TokenDetails/TokenDetails.tsx

```diff
@@ -480,6 +480,13 @@ export function TokenDetails() {
   const navigate = useRainbowNavigate();
   const token = userAsset || customAsset;
 
+  useEffect(() => {
+    const app = document.getElementById('app');
+    setTimeout(() => {
+      app?.focus();
+    }, 150);
+  }, []);
```

Comment:
i guess this was as last resource?

Reply:
Sort of. This issue exists throughout the app, but I placed a [hook](https://github.com/rainbow-me/browser-extension/blob/master/src/design-system/components/AnimatedRoute/AnimatedRoute.tsx#L226) inside `AnimatedRoute` to handle it.

This screen is not using animated route, so I had to put it in manually.